src/Uno.UI/UI/Xaml/Media/LinearGradientBrush.wasm.cs

```diff
@@ -2,7 +2,7 @@
 using System.Linq;
 using Windows.Foundation;
 using Uno.Extensions;
-using Windows.UI.Xaml.Wasm;
+using static Windows.UI.Xaml.Shapes.Shape;
```

Comment:
why this static using?

Reply:
`Windows.UI.Xaml.Wasm.SvgElement` is now
`Windows.UI.Xaml.Shapes.Shape+SvgElement` (a nested class of SvgElement).

It was moved in an _attempt_ to hide it. It is awkward because we need for:
`abstract SvgElement Shape::GetMainSvgElement`
But, at the same time, this doesn't align with UWP api, so it needs to be hidden.

Reply:
There's no need to make it nested, just internal. Is there a problem with hiding it ?

Reply:
I don't see the point in hiddin it.  It could have value for wasm applications even if it's not in the UWP namespace.  I suggest to move it to a `Uno.UI` namespace instead of hidding it.

Reply:
It turns out it's impossible in practice to inherit from `Shape` in UWP (see my comment below), so there's not much value in exposing it.

Reply:
@jeromelaban There was one:
```
internal protected abstract SvgElement Shape.GetMainSvgElement()
```
And, by making `SvgElement` internal, the compiler will complain that:
> CS0050 Inconsistent accessibility: return type 'SvgElement' is less accessible than method 'Shape.GetMainSvgElement()'

The nested internal protected was what I found out to work around this limitation. I guess @carldebilly's suggestion will be a better approach, essentially reverting my last commit.

Reply:
Would `Uno.UI.Xaml.Shapes.SvgElement` be a good place for this?